tests/integration_tests/tests/agentless_tests/test_resume.py

```diff
@@ -329,3 +335,44 @@ def test_force_resume_cancelled_nonresumable(self):
                         .runtime_properties['resumed'])
         self.assertTrue(self.client.node_instances.get(instance2.id)
                         .runtime_properties['marked'])
+
+    def test_restart(self):
```

Comment:
test_resume_failed_restart

Reply:
nothing is failed here... and "resume" is unneeded because it's already in the class name

Reply:
oh i guess `test_restart_resume` is fine, makes for easier `pytest -k`